src/InfoViz/Core/SelectionProvider/histogram2d.js

```diff
@@ -2,19 +2,128 @@
 // Histogram2d
 // ----------------------------------------------------------------------------
 //
+// ===> SET
+//
 //  const payload = {
 //    type: 'histogram2d',
 //    data: {
 //      x: 'temperature',
 //      y: 'pressure',
-//      bins: [...]
-//    }
+//      bins: [...],
+//      score: 0,
+//      annotationId: 24, // Generation of the annotation
+//      selectionId: 23, // Generation of the selection
+//    },
 //  }
 //
+// ===> GET
+//
 //  const query = {
 //    type: 'histogram2d',
-//    axes: ['temperature', 'pressure']
+//    axes: ['temperature', 'pressure'],
+//    score: [0, 2],
 //  }
+//
+// const response = [
```

Comment:
The reason of that duplication (x, y, annotationId, selectionId) is that each item in the response is a self contained histogram2d instead of adding a data conversion at the store time and the usage time. And the ```lot``` of duplication is somewhat arguable ;-).